Source/WebCore/platform/graphics/transforms/TransformationMatrix.cpp

```diff
@@ -907,6 +908,13 @@ TransformationMatrix& TransformationMatrix::scale3d(double sx, double sy, double
     return *this;
 }
 
+static double roundEpsilonToZero(double val)
```

Comment:
Should this use `areEssentiallyEqual()` in MathExtras.h?

Reply:
Probably, yes :) Will change that!

Reply:
Ah nope, it doesn't consider them close enough to be equal :(